### ISSUE TITLE
BottomSheetScrollView animation bug

### ISSUE THREAD

**Issue by boazpoolman** [gorhom/react-native-bottom-sheet#27]
Hi!

First of al; thanks for making this awesome package. It works very well.
Today I tried out the `<BottomSheetScrollView />` component to enhance the scroll behavior in the bottom sheet.
I have 3 snap points, and if I scroll between snap points 1 and 2 everything works fine, but when I scroll to snap point 0 and then scroll back, the animation is not smooth anymore. It just snaps in to place. I made a video to clarify. You can see the animation is working properly untill I swipe the sheet to the bottom.

![issue](https://user-images.githubusercontent.com/9551934/92579553-a32a7280-f28d-11ea-9d58-77b612fba46d.gif)

## Environment info

| Library                         | Version |
| ------------------------------- | ------- |
"@gorhom/bottom-sheet": "^1.2.0",
"react-native": "0.63.0",
"react-native-reanimated": "^1.13.0",
"react-native-gesture-handler": "^1.7.0",

## Reproducible sample code

This is the code for the bottom sheet

```
     const bottomSheetRef = useRef(null);
     const snapPoints = useMemo(() => ['5%', '65%', '80%'], []);
     ...
      <BottomSheet
        ref={bottomSheetRef}
        initialSnapIndex={1}
        snapPoints={snapPoints}
        handleComponent={() => <SheetHandle />}
      >
        <BottomSheetScrollView
          style={{ flex: 1, backgroundColor: 'white' }}
        >
          <View style={{ padding: 20, paddingTop: 0 }}>
            <Text>Lorem...</Text>
          </View>
        </BottomSheetScrollView>
      </BottomSheet>
```

**Comment by gorhom** [comment]
thanks @boazpoolman for reporting this issue, I'll look into it tonight 👍

**Comment by gorhom** [comment]
resolved in `v1.2.1`
